src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer starts working!!!");
+    }
+    @Override
```

Comment:
```suggestion
    }
    
    @Override
```
методу потрібно розділяти пустою строкою для читаймості, спробуй запустити в себе в терміналі команду mvn clean package  - вона створить ріпорт по оформленню коду (чекстайл)

Reply:
Дякую!)